objgraph.py

```diff
@@ -65,6 +65,15 @@
     iteritems = dict.items
 
 
+def _isinstance(object, classinfo):
+    """Use an implementation of isinstance that doesn't depend on ``__class__``,
+    which is delegated to the mock spec by the mocking library."""
```

Comment:
I try to follow [PEP-257](https://www.python.org/dev/peps/pep-0257/) recommendations for docstrings:

- 1st line is a summary, phrased as a command ("Return whether an object is an instance of a class or subclass.")
- 2nd line is blank
- further lines elaborate ("Differs from the builtin isinstance() in that it cannot be fooled by mock.Mock(spec=...)")
- the closing `"""` are on a separate line